src/quick-lint-js/parse.h

```diff
@@ -1538,9 +1538,18 @@ class parser {
           // }
           v.visit_property_declaration(property_name);
         } else {
-          this->error_reporter_->report(error_unexpected_token{
-              .token = property_name_span,
-          });
+          // class C {
+          //   const field
+          // }
+          if ("const" == to_string(property_name_span.string_view())) {
```

Comment:
to_string can allocate memory and is expensive here. Write this instead:
```suggestion
          if (u8"const" == property_name_span.string_view()) {
```